modules/serverless-logic-upgrade-scaling-up-workflows-with-gitops-profile.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-upgrading-operator-from-1.34-to-1.35
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-logic-upgrade-scaling-up-workflows-with-gitops-profile_{context}"]
+= Scaling up workflows with the `gitops` profile
+
+Workflows with the `gitops` profile that were previously scaled down must be scaled back up to continue operation.
+
+.Procedure
+
+. Modify the `my-workflow.yaml` CRD and scale up each workflow to `one` before upgrading as shown in the following example:
++
+[source,yaml]
+----
+spec:
+  podTemplate:
+    replicas: 1
+----
+
+. Apply the updated CRD by executing the following command:
++
+[source,terminal]
+----
+$ oc apply -f <my-workflow.yaml> -n <target_namespace>
+----
+
+. (Optional) Scale the workflow back to `1` by executing the following command:
++
+[source,terminal]
+----
+$ oc patch SonataFlow <my-workflow> -n <target_namespace> --type=merge -p '{"spec": {"podTemplate": {"replicas": 1}}}'
```

Comment:
Is `SonataFlow` (capitalized) correct? Usually CRDs are case-insensitive in oc/kubectl and should be plural (`sonataflows`) unless there's an alias defined in the CRD.

Reply:
@gmunozfe, @wmedvede In the [midstream docs](https://deploy-preview-124--jazzy-shortbread-5f62b7.netlify.app/docs/latest/serverless-logic/cloud/operator/upgrade-serverless-operator/upgrade_1_34_0_to_1_35_0), I saw it is mentioned as "SonataFlow" CRD Can you please let me know which one is correct here?